lib/channel.rb

```diff
@@ -0,0 +1,30 @@
+require "httparty"
+require "dotenv"
+require "pry"
+Dotenv.load
+
+class Channel
```

Comment:
This class is just one class method.  Instead I suggest having a `Channel` class with instance variables and methods.  Your `list` method could then return an array of Channel instances.